cobalt/site/docs/development/setup-android.md

```diff
@@ -27,32 +27,32 @@ return and complete the following steps.
     git checkout tags/24.lts.40
     ```
 
-1. Enter the top-level of the Cobalt directory
+2. Enter the top-level of the Cobalt directory
```

Comment:
I'll revert that part.

I updated it because g3doc recommends using "correct" numbering. Markdown supports both ways and I didn't see any preference in the developer site docs. The developer site markdown formatter don't seem to touch them unlike the regular g3doc markdown formatter.

Reply:
We'll do it in post. We are merging as many ready to go PRs for now as possible to switch to Chrobalt